pkg/signing/signing.go

```diff
@@ -223,6 +367,70 @@ func SifAddSignature(sinfo *image.Sifinfo, signature []byte) error {
 	return nil
 }
 
+func sykeysFetchPubkey(fingerprint string, sykeysAddr string) (openpgp.EntityList, error) {
+	url := "http://" + sykeysAddr + "/pks/lookup?op=get&options=mr&search=0x" + fingerprint
```

Comment:
I'd recommend using a `url.Values` structure for the query params, and a `url.URL` to build up the URL itself. That way, you do not have to worry about whether the params are properly URL encoded. For example:

	v := url.Values{}
	v.Set("op", "get")
	v.Set("options", "mr")
	v.Set("search", "0x" + fingerprint)
	u := url.URL{
		Scheme: "http",
		Host:   sykeysAddr,
		Path:   "pks/lookup",
		RawQuery: 	v.Encode(),
	}

And then you can do `http.Get(url.String())`.

Reply:
Will do !